db/ignore_patterns/global.json

```
@@ -132,15 +132,19 @@
         "^https?://digg\\.com/submit\\?",
         "^https?://(www\\.)?facebook\\.com/(plugins/like(box)?\\.php|sharer/sharer\\.php|sharer?\\.php|dialog/(feed|share))\\?",
         "^https?://www\\.facebook\\.com/captcha/",
-        "^https?://(www\\.)?twitter\\.com/(share\\?|intent/((re)?tweet|favorite)|home/?\\?status=|\\?status=)",
+        "^https?://(www\\.)?twitter\\.com/(share\\?|intent/((re)?(tweet|post)|favorite)|home/?\\?status=|\\?status=)",
         "^https?://platform\\d?\\.twitter\\.com/widgets/tweet_button.html\\?",
         "^https?://www\\.newsvine\\.com/_wine/save\\?",
         "^https?://www\\.netvibes\\.com/subscribe\\.php\\?",
         "^https?://add\\.my\\.yahoo\\.com/(rss|content)\\?",
         "^https?://www\\.addtoany\\.com/(add_to/|share_save\\?)",
         "^https?://www\\.addthis\\.com/bookmark\\.php\\?",
         "^https?://(www\\.)?pinterest\\.com/pin/create/",
-        "^https?://www\\.linkedin\\.com/(cws/share|shareArticle)\\?",
+        "^https?://(www\\.)?linkedin\\.com/(cws/share|shareArticle|sharing/share-offsite/)\\?",
+        "^https://gettr\\.com/share\\?",
+        "^https://telegram\\.me/share/url\\?",
```

```suggestion
        "^https://(telegram|t)\\.me/share/url\\?",
```

not sure if I've seen http in the wild, but it might be worth including too

---

Added, and also added http/https www/non-www.